code-style/git/README.md

```diff
@@ -208,15 +208,24 @@ Every place you work will have a different Git flow. At Sparkbox the Git flow is
 
     -  Protip: Sometimes we create pull requests early as a place to collaborate on a solution. Consider labeling these as `DO NOT MERGE` or `WIP`.
```

Comment:
Since you are in this file, you could add a sentence at the end this linking to 

```md

You may also utilize [Draft PRs][].
```
(add this to links) 
```md
[Draft PRs]: https://github.blog/2019-02-14-introducing-draft-pull-requests/
```